CHANGELOG.rst

```diff
@@ -1,3 +1,7 @@
+**5.0.0 - 06/02/25**
+
+  - Feature: Update LBWSG to use exposure data based on simulant age
```

Comment:
This is not reverse compatible so should require a major release but I think the description here could be better